src/qt/signverifymessagedialog.cpp

```diff
@@ -246,6 +246,21 @@ void SignVerifyMessageDialog::on_verifyMessageButton_VM_clicked()
             QString("<nobr>") + tr("Message verification failed.") + QString("</nobr>")
         );
         return;
+    case MessageVerificationResult::INCONCLUSIVE:
+        ui->statusLabel_VM->setText(
+            tr("Message inconclusive (your node may need to be upgraded to validate new upgradable rules).")
```

Comment:
This might need rephrasing to avoid an attack where someone uses this to trick a user into "upgrading" to malware...